### PR TITLE
Add md to html converter (TOOLS-183)

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Added markdown to html convertor (uses https://github.com/marketplace/actions/markdown-to-pdf-and-html)
   (Styles coud be added through css file)
 - Changed destination of document files to `docs` in ize.sh repo
 
#### Testing done: 
screenshot 2022-02-10 at 20 13 00

### CODE REVIEW

File: .github/workflows/build_on_release.yml [hunk 1]
```diff
@@ -41,14 +41,27 @@ jobs:
...
+        input_dir: commands
+        output_dir: docs
+        # Default is true, can set to false to only get PDF files
+        build_html: true
+
+    - name: remove pdfs #This needs because action creates PDF by default
+      run: |
+        cd docs
+        sudo rm *.pdf
+
```
Comment (AutomationD) [inline .github/workflows/build_on_release.yml:56]: What happens when we go ize.sh/docs?  I'd say we should rename ize.html to index.html, no?

File: .github/workflows/build_on_release.yml [hunk 2]
```diff
@@ -41,14 +41,27 @@ jobs:
         mkdir -p commands
         go run ./cmd gendoc
 
+    - name: convert MD to HTML
```
Comment (AutomationD) [inline .github/workflows/build_on_release.yml:44]: How do we provide styling? Can you attach tailwind CSS to the style? The CSS is available here https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, see comments

File: .github/workflows/build_on_release.yml (hunk 1 above)
Comment (kobrikx) [reply .github/workflows/build_on_release.yml:59]: Renamed ize.html to index.html and rename links in html files

**Pull request merged**